rand_distr/src/poisson.rs

```diff
@@ -147,10 +147,27 @@ where Standard: Distribution<N>
     }
 }
 
+impl<N: Float> Distribution<usize> for Poisson<N>
+    where Standard: Distribution<N>
+{
+    #[inline]
+    fn sample<R: Rng + ?Sized>(&self, rng: &mut R) -> usize {
+        let result: u64 = self.sample(rng);
+        result as usize
```

Comment:
This would have a zero probability of values > `u64::MAX` for 128bit architectures.

Reply:
Yes, that would be very bad. I changed it to include a `to_usize`. Is this better?